qubesmanager/utils.py

```diff
@@ -49,6 +52,25 @@
 # - just a list, no properties or defaults, just a nice list with a "current"
 #       value - initialize_widget
 
+def set_organization():
+    """set the organization and config home directory for Qt based Apps."""
+    # migrate old config file to the unified config directory
+    # TODO: remove the below block when Qubes R4.3 becomes End-of-Life
+    old_config_home = xdg.BaseDirectory.xdg_config_home + \
+        '/The Qubes Project'
+    new_config_home = xdg.BaseDirectory.xdg_config_home + '/qubes-os'
```

Comment:
I believe it is better to derive the new configuration directory from the `OrganizationName` to avoid code duplication.

Reply:
This is not possible. Since the config file copy operation (if needed) happens before the QtCore initialization. This is done intentionally to avoid any conflict with QtCore mechanism. Just to be on the safe side.